spec.yaml

```diff
@@ -30,13 +30,24 @@ parameters:
       the HAL navigation links (e.g. `_links.next.href`) to paginate. These
       links enable the possibility to use vendor specific pagination.
 paths:
-  /ocm-provider:
+  /.well-known:
```

Comment:
This should probably be /.well-known/ocm-provider or something: https://en.wikipedia.org/wiki/Well-known_URI

Reply:
That's right, when I drafted it I just forgot that `/.well-known` lives at the top level and we need a sub path.

By the way, the spec was unclear about the discovery endpoint being served as a top-level URL, therefore I added a specific line about that.